server/src/main/java/org/elasticsearch/action/admin/indices/shards/TransportIndicesShardStoresAction.java

```diff
@@ -279,5 +225,59 @@ public class Response {
                 }
             }
         }
+
+        void finish() {
```

Comment:
This method had no business being a member of `InternalAsyncFetch`, this change just moves it up a level.